10/umbraco-cms/implementation/controllers.md

```diff
@@ -52,8 +52,34 @@ An Umbraco Authorized Controller is used when the controller requires member or
 The Umbraco Authorized controllers and attributes for Backoffice Users are:
 
 *   **MVC**
+    There is no specific controller available to inherit from. We recommend inheriting from the basic `Umbraco.Cms.Web.Common.Controllers.UmbracController`` and applying the following attributes to your method
```

Comment:
checking https://apidocs.umbraco.com/v13/csharp/api/Umbraco.Cms.Web.Common.Controllers.UmbracoController.html it seems to be `Umbraco.Cms.Web.Common.Controllers` instead of `Umbraco.Cms.Web.Common.Controllers.UmbracController` can you please confirm as well? 😊 
```suggestion
    There is no specific controller available to inherit from. We recommend inheriting from the basic `Umbraco.Cms.Web.Common.Controllers.Controllers` and applying the following attributes to your method:
```